Foreman Integration Pack

nullkarma opened the pull request.

LindsayHill commented:
https://github.com/StackStorm-Exchange/stackstorm-foreman now created (content not yet transferred)

nullkarma commented:
Hi Lindsay. I fixed the 100 chars line length, added the icon.png, but didn't resolve the weaker warnings like unused local vars (*args, **kwargs). If you want me to remove those too, i'll do it.
Feel free to transfer the content.

LindsayHill commented:
Thanks for that. It is just the more critical issues that need resolving, like 100-char lines. Once this gets transferred to its own repo, new PRs will trigger CircleCI checks, which include linting. Fixing it now saved me a step later :)
